src/scss/components/selector-strip/_hxSelector.scss

```diff
@@ -56,41 +59,53 @@
       // unchecked (focused)
       &:focus + span {
         background-color: $cyan-100;
+        background-color: var(--hxSelector-unchecked-bgcolor, $cyan-100);
         color: $cyan-900;
+        color: var(--hxSelector-unchecked-color, $cyan-900);
```

Comment:
Please use updated  `v2.0` colors to replace `$cyan-XXX`.

Reply:
Changes have been done, please review.